docs/java/how-to-develop-a-workflow-definition-in-java.md

```diff
@@ -16,18 +19,15 @@ public class FileProcessingWorkflowImpl implements FileProcessingWorkflow {
 ## Workflow interface
 
 The Workflow interface is a Java interface and is annotated with `@WorkflowInterface`.
-Workflow interface methods must have one of the following annotations:
+Workflow interface methods must have one [`@WorkflowMethod`](#workflowmethod).
+Use `@SignalMethod`, `@QueryMethod` for Signals, and Queries in the Workflow.
```

Comment:
Not quite sure how to read this sentence.

Reply:
lol yeah. changed to:
"The Workflow interface is a Java interface and is annotated with `@WorkflowInterface`.
Workflow interface methods must have one [`@WorkflowMethod`](#workflowmethod).
Use `@SignalMethod` for Signals, and `@QueryMethod` for Queries in the Workflow."